ProceduralForest/Structures/mesh.h

```diff
@@ -0,0 +1,139 @@
+#ifndef MESH_H
```

Comment:
There seems to be a duplicate of this file, although it doesnt seem to be ever imported. It is also a direct copy of what is found at https://learnopengl.com/code_viewer_gh.php?code=includes/learnopengl/mesh.h

without given any attribution to the author or source.

Reply:
yeah, it was taken directly from it, i havent included the attribution to the author